DataDefinitions/StationModel.cs

```diff
@@ -45,8 +47,9 @@ private StationModel(string edname) : base(edname, edname.Replace(" Starport", "
             Dictionary<string, string> modelTranslations = new Dictionary<string, string>()
             {
                 { "Coriolis Starport", "Coriolis" },
-                // Ocellus starports are described by the journal as "Bernal"
-                { "Ocellus Starport", "Bernal" }, 
+                // Ocellus starports are described by the journal as either "Bernal" or "Ocellus"
+                { "Bernal Starport", "Bernal" },
```

Comment:
Are we still seeing 'Bernal'? I thought they had switched over to 'Ocellus' completely.

Reply:
As of when? Here's my latest journal with a 'Bernal' station model:
```json
{ "timestamp":"2019-03-21T02:59:35Z", "event":"Location", "Docked":true, "StationName":"Explorer's Anchorage", "StationType":"Bernal", "MarketID":128927917, "StationFaction":{ "Name":"Deep Space Surveys", "FactionState":"Investment" }, "StationGovernment":"$government_Democracy;", "StationGovernment_Localised":"Democracy", "StationServices":[ "Dock", "Autodock", "BlackMarket", "Commodities", "Contacts", "Exploration", "Missions", "Outfitting", "CrewLounge", "Rearm", "Refuel", "Repair", "Tuning", "Workshop", "MissionsGenerated", "FlightController", "StationOperations", "SearchAndRescue", "StationMenu" ], "StationEconomy":"$economy_Extraction;", "StationEconomy_Localised":"Extraction", "StationEconomies":[ { "Name":"$economy_Extraction;", "Name_Localised":"Extraction", "Proportion":1.000000 } ], "StarSystem":"Stuemeae FG-Y d7561", "SystemAddress":259804475954059, "StarPos":[28.68750,-19.78125,25899.68750], "SystemAllegiance":"Independent", "SystemEconomy":"$economy_Extraction;", "SystemEconomy_Localised":"Extraction", "SystemSecondEconomy":"$economy_None;", "SystemSecondEconomy_Localised":"None", "SystemGovernment":"$government_Democracy;", "SystemGovernment_Localised":"Democracy", "SystemSecurity":"$SYSTEM_SECURITY_medium;", "SystemSecurity_Localised":"Medium Security", "Population":750000, "Body":"Explorer's Anchorage", "BodyID":16, "BodyType":"Station", "Factions":[ { "Name":"Pilots Federation Local Branch", "FactionState":"None", "Government":"Democracy", "Influence":0.000000, "Allegiance":"PilotsFederation", "Happiness":"", "MyReputation":100.000000 }, { "Name":"Deep Space Surveys", "FactionState":"Investment", "Government":"Democracy", "Influence":0.501000, "Allegiance":"Independent", "Happiness":"$Faction_HappinessBand2;", "Happiness_Localised":"Happy", "MyReputation":100.000000, "ActiveStates":[ { "State":"Investment" }, { "State":"PirateAttack" } ] }, { "Name":"Transorbital Dynamics", "FactionState":"None", "Government":"Democracy", "Influence":0.128000, "Allegiance":"Independent", "Happiness":"$Faction_HappinessBand2;", "Happiness_Localised":"Happy", "MyReputation":0.000000 }, { "Name":"Mishka Corp.", "FactionState":"None", "Government":"Democracy", "Influence":0.131000, "Allegiance":"Independent", "Happiness":"$Faction_HappinessBand2;", "Happiness_Localised":"Happy", "MyReputation":0.000000 }, { "Name":"Dawnlight Core Mining Operations", "FactionState":"None", "Government":"Democracy", "Influence":0.110000, "Allegiance":"Independent", "Happiness":"$Faction_HappinessBand2;", "Happiness_Localised":"Happy", "MyReputation":0.000000 }, { "Name":"The Order of Imagination", "FactionState":"None", "Government":"Democracy", "Influence":0.130000, "Allegiance":"Independent", "Happiness":"$Faction_HappinessBand2;", "Happiness_Localised":"Happy", "MyReputation":0.000000 } ], "SystemFaction":{ "Name":"Deep Space Surveys", "FactionState":"Investment" } }
```

Reply:
March 21st is ancient. Pretty sure 'Bernal' is dead. We can check with EDCD.

Reply:
A quick search reveals no public comments about this if it has changed. If you're sure, maybe you saw something about it in consul chat?

Reply:
Asking for you. Meanwhile it's no reason the hold things up.